packages/labs/test/timezonePickerTests.tsx

```diff
@@ -209,6 +209,11 @@ describe("<TimezonePicker>", () => {
         });
     });
 
+    it("manual metadata matches moment metadata", () => {
```

Comment:
👌 

Would be helpful to have a test where the two aren't equal (i.e. a "less accurate" case).

Reply:
@cmslewis had a discussions with @giladgray about possibly removing the forking logic and doing a examination about whether or not it is actually less accurate.

Specifically, I actually don't know of any case where they aren't equal but have not run a comprehensive test checking

Reply:
@cmslewis @giladgray wrote up a fiddle here to test, with `{ +1, 0, +1 } X untils` per timezone and tested against all timezones

http://jsfiddle.net/9hnsLgx6/5/ 

Looks like there are 2338 unique dates, 583 timezones, 1362471 matches, 0 misses. 

If this looks good to just replace the logic, I can remove the forking behavior?